src/pages/Router.js

```diff
@@ -0,0 +1,18 @@
+import { render } from '../shared/dom';
+
+export const Router = ({ routes, on }) => {
```

Comment:
Just two more things and you are good to go 8)

create routes.js file with empty object like so with one example
`export const routes = {
home: 'home'
}`

In comment define how the routes object should look like (above the Router component ifselt)